### DISCUSSION TITLE
Fixed or Min/Max Y Axis Range

### DISCUSSION THREAD

**Discussion by jllamas** [nasa/openmct#7677] [Community Q&A] [answered]
Hello,

We are running:
```
Version: 2.2.1
Build Date: Mon Apr 03 2023 11:46:05 GMT-0700 (Pacific Daylight Time)
Revision: 3c60b0416f004f213bf5c1642c76b02a35408460
Branch: release/2.2.1
```

We are curious if there is a way to fix or set a min/max for the Y axis instead of it being dynamic?

As an example, set a soft upper limit to 100 for a chart. The Y range should be 0-100. Then if a value goes over 100 the chart can dynamically adjust.

Thanks in advance,
Jake

**Comment by ozyx** [comment, marked as answer]
Hi @jllamas ,

Thanks for reaching out! Currently, in Open MCT, you can set fixed min and max values for the Y-axis in Overlay Plots: 

https://github.com/nasa/openmct/assets/9259993/32d0f54e-75ee-446c-bc55-2b0768adcc75

Once these limits are set, they remain fixed, even if subsequent values exceed these bounds. It sounds like you're interested in a feature where the Y-axis could automatically revert to autoscaling when values surpass the predefined limits. We don't support this functionality today, but if you see value in it, we encourage you to open an [enhancement request](https://github.com/nasa/openmct/issues/new?assignees=&labels=type%3Aenhancement&projects=&template=enhancement-request.md&title=).

Cheers,
Jesse
